examples/image_i2c.rs

```diff
@@ -75,7 +74,7 @@ fn main() -> ! {
     disp.init().unwrap();
     disp.flush().unwrap();
 
-    let im = Image1BPP::new(include_bytes!("./rust.raw"), 64, 64).translate(Point::new(32, 0));
+    let im = ImageLE::new(include_bytes!("./rust.raw"), 64, 64).translate(Point::new(32, 0));
```

Comment:
For <= 8bpp images you don't need to explicitly specify the byte order and can simply use `Image` instead.

Reply:
I get an error about being unable to resolve `BO` if I use `Image` over `ImageLE` here. I could specify the type on `im` but it's cleaner to just use `ImageLE`, no?

Reply:
I didn't realize that there was no type on `im`.

I prefer to explicitly set the type for images, because this specifies the data type stored in the image. If no color type is specified the compiler will infer some type, which probably matches the display, but this might not be correct for the image.

Reply:
I've moved both image examples over to the following.

```rust
let im: Image<BinaryColor> = ...
```

Retested the `image_i2c` example and it still works (as expected).